### PR TITLE
Update old propType methods

### PR DESCRIPTION
**Pull request by MarioRozic**
Hey I used your module and it's relay nice but some code was outdated, so I decided to update it so other people don't have to worry about it. 

Edit: I see my editor did some formating, hope you don't mind that.

### CODE REVIEW

File: SearchList.js [hunk 1]
```diff
@@ -105,33 +79,33 @@ export default class SearchList extends Component {
     this.tmpSource = []
   }
 
-  static getSectionData (dataBlob, sectionID) {
+  static getSectionData(dataBlob, sectionID) {
```
Comment (erichua23) [inline SearchList.js]: Our code style follows the [standardjs](https://standardjs.com/), fix the code format please, thx.

File: SearchList.js [hunk 2]
```diff
@@ -634,6 +608,33 @@ export default class SearchList extends Component {
   }
 }
 
+SearchList.propTypes = {
```
Comment (erichua23) [inline SearchList.js]: Your move all static properties out of the class, do your have any problem when using static properties?

Comment (MarioRozic) [reply SearchList.js]: Nope, its just cleaner this way, at least for me, I can put them back in class if u want but its the same thing..

File: SearchList.js (hunk 1 above)
Comment (MarioRozic) [reply SearchList.js]: ok ok I will

**Pull request merged**